studentRoutes.js

```diff
@@ -0,0 +1,126 @@
+const router = require('express').Router();
+const knex = require('knex'); 
+
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+      filename: './data/lambda.sqlite3',
+    },
+    // debug: true,
```

Comment:
put the config object in your knexfile to good use! you can write your paths and other properties on it there instead of declaring in both routes